src/main/java/org/session4/homework/Triangle.java

```diff
@@ -0,0 +1,13 @@
+package org.session4.homework;
+
+public class Triangle {
+
+//    Write a Java program that determines the type of a triangle based on its sides.
+//    Declare three variables sideOne, sideTwo and sideThree to represent the sides of the triangle.
+//    Use the ternary operator to determine and print whether the triangle is "Equilateral", "Isosceles", or "Scalene".
+
```

Comment:
Please remove this empty row, we don't need to highlight something here. When we can save some space, we will do it.

Reply:
oh, all these lines where something I forgot to clean up before commiting.